pkg/apis/kops/validation/legacy.go

```diff
@@ -248,11 +248,13 @@ func ValidateCluster(c *kops.Cluster, strict bool) error {
 
 	// AdminAccess
 	if strict && len(c.Spec.SSHAccess) == 0 {
+		// TODO: We may want to allow this
 		return fmt.Errorf("SSHAccess not configured")
 	}
 
 	// AdminAccess
 	if strict && len(c.Spec.KubernetesAPIAccess) == 0 {
+		// TODO: We may want to allow this (maybe)
```

Comment:
It isn't very useful to have an API server that allows no access?  But I can maybe imagine it.

OTOH nobody has ever asked for it.